components/EditCoverSection.tsx

```diff
@@ -56,27 +53,29 @@ const EditCoverSection = () => {
     setHome({ ...home, [name]: value });
   };
 
-  const handleOpen = (name: string): void => {
-    setIsOpen(isOpen === name ? null : name);
+  const imageRef = React.useRef<HTMLImageElement>(null);
+  const imageRef2 = React.useRef<HTMLImageElement>(null);
+  const imageRef3 = React.useRef<HTMLImageElement>(null);
+
+  const [isOpen, setIsOpen] = useState<
+    RefObject<HTMLImageElement> | undefined | null
+  >(null);
+
+  const [isOverlay, setIsOverlay] = useState<boolean>(false);
```

Comment:
`isOverlay` state seems redundant, why can't we use `isOpen`?

Reply:
right, thanks :)